heltour/tournament/tests/test_views.py

```diff
@@ -84,7 +111,12 @@ def test_template(self):
         response = self.client.get(season_url('team', 'crosstable'))
         self.assertTemplateUsed(response, 'tournament/team_crosstable.html')
 
-        response = self.client.get(season_url('lone', 'crosstable'))
+        # triggering a 404 writes to the log, disable that temporarily for nicer test output
+        logging.disable(logging.CRITICAL)
```

Comment:
Ok, yeah, with the heavy use, you should consider a context manager to disable logs so you can do:
```
with noLogging():
    doSomething()
```